crates/matrix-sdk-crypto/src/olm/mod.rs

```diff
@@ -220,20 +222,9 @@ pub(crate) mod tests {
             "room_id": room_id,
             "type": "m.room.encrypted",
             "content": encrypted_content,
-        })
-        .to_string();
-
-        let event: AnySyncRoomEvent = serde_json::from_str(&event).unwrap();
-
-        let event = if let AnySyncRoomEvent::MessageLike(AnySyncMessageLikeEvent::RoomEncrypted(
-            SyncMessageLikeEvent::Original(event),
-        )) = event
-        {
-            event
-        } else {
-            panic!("Invalid event type")
-        };
+        });
 
+        let event = Raw::new(&event).unwrap().deserialize_as().unwrap();
```

Comment:
Can be `json_convert`.

Reply:
Found another one but I think I won't spend more time searching for those patterns currently.